src/components/button/Button.scss

```
@@ -0,0 +1,138 @@
+@import '../../styles/colors';
+
+.summary {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 150px;
+  height: 50px;
+  left: 1197px;
+  top: 95px;
+  color: $white;
+  background-color: $orange;
+  border-radius: 30px;
+
+  .divider {
+    padding: 0 15px;
+  }
+
+  .cart-container {
+    position: relative;
+    margin-bottom: 5px;
+  }
+
+  .cart {
+    padding: 0 5px 0 0;
+  }
+}
+
+.filter {
+  margin: 10px 10px 10px 0;
+  top: 175px;
+  padding: 10px 28px;
+  box-sizing: content-box;
+  border-radius: 30px;
+}
+
+.add {
+  width: 132px;
+  color: $white;
+  background-color: $orange;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 6px;
+  height: 40px;
+  box-shadow: 0 15px 20px rgba(0, 0, 0, 0.03);
+
+  .add-button-name {
+    color: white;
+    font-size: 15px;
+    font-family: sans-serif;
+  }
+}
+
+.added {
+  width: 153px;
+  border: 1px solid $orange;
+  background-color: $white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 7px;
+  height: 40px;
+  box-shadow: 0 15px 20px rgba(0, 0, 0, 0.03);
+
+  .add-button-name {
+    color: $orange;
+    font-size: 15px;
+    font-family: sans-serif;
+  }
+
+  .notification-round-container {
+    position: relative;
+    display: inline-block;
+    width: 20px;
+    height: 20px;
+  }
+
+  .notification-container-text {
+    position: absolute;
+    display: block;
+    height: 20px;
+    width: 20px;
+    color: #fff;
+    background-color: $orange;
+    border-radius: 22px;
+    font-size: 10px;
+    font-weight: 700;
+    line-height: 21px;
+    text-align: center;
+  }
+}
+
+.back {
+  display: inline-flex;
+  align-items: center;
+  width: auto;
+  height: 45px;
+  padding: 0 25px;
+  margin-bottom: 25px;
+  color: $white;
+  background-color: $white;
+  border: 1px solid $light-grey;
+  border-radius: 30px;
+  gap: 10px;
+
+  .button-back-title {
+    font-weight: 400;
+    color: $light-grey;
+  }
+}
+
+.pay {
+  height: 45px;
+  margin: 0 18px;
+  padding: 0 18px;
+  background-color: $orange;
+
+  .button-pay-title {
+    padding: 0 14px;
+    color: $white;
+    font-weight: 500;
+  }
+}
+
+.backBlack {
```

не надо смешивать разные стили именования 
```suggestion
.back-black {
```